src/Native/Unix/System.Security.Cryptography.Native/pal_ssl.h

```diff
@@ -398,3 +398,6 @@ Looks up a cipher by the IANA identifier, returns a shared string for the OpenSS
 and emits a value indicating if the cipher belongs to the SSL2-TLS1.2 list, or the TLS1.3+ list.
 */
 DLLEXPORT const char* CryptoNative_GetOpenSslCipherSuiteName(SSL* ssl, int32_t cipherSuite, int32_t* isTls12OrLower);
+
+/* Internal APIs */
+void ResetProtocolRestrictions(SSL_CTX* ctx);
```

Comment:
I don't see any need for this to be in the header.

Reply:
actually I'm getting warnings which we treat as errors without it (that's why I added it in the first place). Will see if there is any way to get around that (I could probably move this to the .c file though)

Reply:
nvm, just read the comment above